### ISSUE TITLE
Support for pinch/zoom events

### ISSUE THREAD

**Issue by jfedyczak** [can-lehmann/owlkettle#173]
Would love to use pinch/zoom events or at least expose dx/dy on scroll event.

**Comment by can-lehmann** [comment]
Are you referring to supporting them on `CustomWidget` (meaning also `DrawingArea` and `GlArea`)? There is already a `scroll` event which should give you smooth scroll values.

**Comment by jfedyczak** [comment]
Yes, I'm using `scroll` event already in `DrawingArea` and it works great. I was hoping for additional events for touch screens (pinch for zooming in/out) - not sure if GTK supports them though.
